src/components/OpenInDesktopAppHandler.tsx

```diff
@@ -33,7 +33,7 @@ export const OpenInDesktopAppHandler = (props: React.PropsWithChildren) => {
   function onOpenInDesktopApp() {
     const newSearchParams = new URLSearchParams(globalThis.location.search)
     newSearchParams.delete(ASK_TO_OPEN_QUERY_PARAM)
-    const newURL = `${ZOO_STUDIO_PROTOCOL}${globalThis.location.pathname.replace(
+    const newURL = `${ZOO_STUDIO_PROTOCOL}://${globalThis.location.pathname.replace(
```

Comment:
This was missing from the web app in order to properly open the app. Might be related to the change in the `ZOO_STUDIO_PROTOCOL` var too.

Reply:
Thank you for figuring this out, I would've missed this.

Reply:
No worries at all, glad we did it step by step. We'll likely find other things before we can make this go live on released versions.